submissions/Jemikalajah-Philip/Html/registration-form.html

```diff
@@ -1,105 +1,100 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 
 <head>
-    <title>Registration Form</title>
-    <link rel="stylesheet" type="text/css" href="../css/registration.css">
+  <title>registration form</title>
+  <link rel="stylesheet" type="text/css" href="../css/style.css" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
 </head>
 
 <body>
-    <div class="main">
-        <div class="topbar">
-
+  <div class="main">
+    <div class="topbar"></div>
+    <div class="social_icon">
+      <div class="container">
+        <div class="icon">
+          <a href="https://facebook.com"><img src="icon/facebook.png" width="30px" height="30px" /></a>
```

Comment:
I think you should create a class for this css style instead of writing them inline.